lambdas/transcode/index.py

```diff
@@ -97,4 +122,13 @@ def lambda_handler(request):
 
         data = output_file.read()
 
-    return 200, data, {'Content-Type': f'video/{format}'}
+    parsed = urlparse(url)
+    filename = parsed.path.rsplit('/', 1)[-1]
```

Comment:
very minor and sorry if I'm being pedantic, but this feels cleaner
```suggestion
    filename = parsed.path.rpartition('/')[-1]
```